internal/tsc_wrapped/compiler_host.ts

```diff
@@ -184,12 +184,32 @@ export class CompilerHost implements ts.CompilerHost, tsickle.TsickleHost {
     return moduleName;
   }
 
+  // Converts file path into a valid AMD module name
+  amdModuleName(sf: ts.SourceFile) {
+    let fileName = sf.fileName;
+    // TypeScript gives us a path with the rootDirs[i] still present, eg.
+    // /build/work/bazel-out/local-fastbuild/bin/path/to/file
+    // We want path/to/file.
+    for (const root of this.options.rootDirs) {
```

Comment:
Isn't there a helper function someehre?

Reply:
No, right now this is duplicated with the function above that creates `goog.module` names.
I extracted a helper for it.